rest/model/data/meta.go

```diff
@@ -179,6 +179,13 @@ func FormatInterface(i interface{}) string {
 			slc = append(slc, s.(string))
 		}
 		return strings.Join(slc, ",")
+	case map[string]interface{}:
+		// Required for Terraform workaround to allow users to submit raw json of feed pointer
+		// as value for metadata.  See https://github.com/terraform-providers/terraform-provider-ns1/issues/35
+		mapVal := i.(map[string]interface{})
```

Comment:
isn't this just the `v` value from above? or is there a reason you are "recasting" `i` here?

Reply:
Good catch, no reason to recast